pkg/runner/artillery.go

```diff
@@ -145,7 +144,7 @@ func CreateEnvFile(vars map[string]testkube.Variable) (*os.File, error) {
 	for _, v := range vars {
 		envVars = append(envVars, []byte(fmt.Sprintf("%s=%s\n", v.Name, v.Value))...)
 	}
-	envFile, err := ioutil.TempFile("/tmp", "")
+	envFile, err := os.CreateTemp("/tmp", "")
```

Comment:
should we return file descriptor or we just need name of the file?

Reply:
Good point, updated the code, hope it looks better now :)